scripts/ci/install_hadoop.sh

```diff
@@ -7,6 +7,8 @@ set -e
 
 sudo apt-get update -y
 sudo apt-get install default-jdk
+# Use this in newer Ubuntu versions - 18.04+
```

Comment:
Isn't default-jdk on xenial openjdk-8-jdk?

Reply:
It's `openjdk-11` in Ubuntu 18.04. And this script is not only for Travis anymore, it's mentioned in docs. I don't know for sure that jdk-11 wouldn't work, however, I had a number of issues with it.

Reply:
@Suor I mean if it is openjdk-8 on xenial too, then we can simply use
```
sudo apt-get install openjdk-8-jdk
```
instead of
```
sudo apt-get install default-jdk
# Use this in newer Ubuntu versions - 18.04+
```

Reply:
Installed Xenial to check it) and it in fact have default-jdk = openjdk-8-jdk